create_table/main.py

```diff
@@ -1,5 +1,4 @@
 from google.cloud import bigquery
-from google.cloud import secretmanager
```

Comment:
Pycharm mistake. Since `secretmanager` was not used in this file, Pycharm erased the import.